server_addon/houdini/server/settings/publish.py

```diff
@@ -77,25 +77,29 @@ class PublishPluginsModel(BaseSettingsModel):
         default_factory=CollectLocalRenderInstancesModel,
         title="Collect Local Render Instances."
     )
-    ValidateInstanceInContextHoudini: BasicValidateModel = SettingsField(
-        default_factory=BasicValidateModel,
+    ValidateInstanceInContextHoudini: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
         title="Validate Instance is in same Context.",
         section="Validators")
-    ValidateMeshIsStatic: BasicValidateModel = SettingsField(
-        default_factory=BasicValidateModel,
+    ValidateMeshIsStatic: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
         title="Validate Mesh is Static.")
-    ValidateReviewColorspace: BasicValidateModel = SettingsField(
-        default_factory=BasicValidateModel,
+    ValidateReviewColorspace: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
         title="Validate Review Colorspace.")
-    ValidateSubsetName: BasicValidateModel = SettingsField(
-        default_factory=BasicValidateModel,
+    ValidateSubsetName: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
         title="Validate Subset Name.")
-    ValidateUnrealStaticMeshName: BasicValidateModel = SettingsField(
-        default_factory=BasicValidateModel,
+    ValidateUnrealStaticMeshName: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
         title="Validate Unreal Static Mesh Name.")
     ValidateWorkfilePaths: ValidateWorkfilePathsModel = SettingsField(
         default_factory=ValidateWorkfilePathsModel,
         title="Validate workfile paths settings.")
+    ExtractActiveViewThumbnail: BasicEnabledStatesModel = SettingsField(
+        default_factory=BasicEnabledStatesModel,
+        title="Extract Active View Thumbnail"
```

Comment:
could we add a section
```suggestion
        title="Extract Active View Thumbnail",
        section="Extractors"
```
how my suggestion looks on my side
![image](https://github.com/ynput/ayon-core/assets/20871534/98e09352-aa4b-4d7b-8916-a8b40ff928d3)